yb-voyager/cmd/importDataTestUtils.go

```diff
@@ -99,7 +99,7 @@ func setupExportDirAndImportDependencies(batchSizeRows int64, batchSizeBytes int
 	return ldataDir, lexportDir, state, nil
 }
 
-func createFileAndTask(lexportDir string, fileContents string, ldataDir string, tableName string) (string, *ImportFileTask, error) {
+func createFileAndTask(lexportDir string, fileContents string, ldataDir string, tableName string, id int) (string, *ImportFileTask, error) {
```

Comment:
how do we use the id here?
Also please add a comment for this function

Reply:
Id is just stored in ImportFileTask.ID. We now use it to find the task in the picker (for example when marking it as done).